routes/books.js

```diff
@@ -0,0 +1,48 @@
+import express from "express";
+import booksData from "../data/books.json";
+
+const router = express.Router()
+
+router.get("/", (req, res) => {
+    res.json(booksData)
+})
+
+router.get("/top-rated", (req, res) => {
+    const highAvgRating = booksData.filter((book) => book.average_rating >= 4)
+    res.status(200).json(highAvgRating)
+})
+
+router.get("/title/:title", (req, res) => {
+    const title = req.params.title
+    let titleInput = booksData.find((str) => str.title.toLowerCase() === title.toLowerCase())
+
+    if (titleInput) {
+        res.status(200).json({
+            data: titleInput,
+            success: true
+        })
```

Comment:
I noticed that when I try to find any of the Harry Potter books, I hit your else statement. I can't seem to be able to hit this endpoint for example: /books/title/Harry Potter and the Order of the Phoenix (Harry Potter  #5)